A test of the work-around using `terraform state rm …` allowed the Terraform plan to successfully be created.
(The Terraform state file `azul--hammerbox.tfstate ` was restored to the previous version `StKw1xh3YusC6.g6Om_qPT2.ml_LW_8R` via the AWS console after this test.)

```
(.venv) daniel@Crispin ~/repo/azul1 $ CI_COMMIT_REF_NAME=anvilprod make deploy
/Library/Developer/CommandLineTools/usr/bin/make -C lambdas

…

Terraform planned the following actions, but then encountered a problem:

  # aws_cloudwatch_dashboard.dashboard will be updated in-place
…
  # aws_cloudwatch_log_group.error_log will be destroyed
  # (because aws_cloudwatch_log_group.error_log is not in configuration)
…
  # aws_cloudwatch_log_group.index_log will be destroyed
  # (because aws_cloudwatch_log_group.index_log is not in configuration)
…
  # aws_cloudwatch_log_group.search_log will be destroyed
  # (because aws_cloudwatch_log_group.search_log is not in configuration)
…
  # aws_cloudwatch_log_resource_policy.index will be destroyed
  # (because aws_cloudwatch_log_resource_policy.index is not in configuration)
…
  # aws_elasticsearch_domain.index will be destroyed
  # (because aws_elasticsearch_domain.index is not in configuration)
…
  # aws_iam_role_policy.indexer will be updated in-place
…
  # aws_iam_role_policy.service will be updated in-place
…
  # aws_lambda_function.indexer will be updated in-place
…
  # aws_lambda_function.indexer_aggregate will be updated in-place
…
  # aws_lambda_function.indexer_aggregate_retry will be updated in-place
…
  # aws_lambda_function.indexer_contribute will be updated in-place
…
  # aws_lambda_function.indexer_contribute_retry will be updated in-place
…
  # aws_lambda_function.indexer_indexercachehealth will be updated in-place
…
  # aws_lambda_function.service will be updated in-place
…
  # aws_lambda_function.service_manifest will be updated in-place
…
  # aws_lambda_function.service_servicecachehealth will be updated in-place
…
  # aws_lambda_layer_version.dependencies must be replaced
…
  # aws_security_group.elasticsearch will be destroyed
  # (because aws_security_group.elasticsearch is not in configuration)
…

Plan: 1 to add, 12 to change, 7 to destroy.
╷
│ Error: Missing required argument
│
│ The argument "url" is required, but was not set.
╵
make[1]: *** [apply] Error 1
make: *** [terraform] Error 2
(.venv) daniel@Crispin ~/repo/azul1 $
(.venv) daniel@Crispin ~/repo/azul1 $ cd terraform/
(.venv) daniel@Crispin ~/repo/azul1/terraform $ terraform state rm opensearch_cluster_settings.index
Removed opensearch_cluster_settings.index
Successfully removed 1 resource instance(s).
(.venv) daniel@Crispin ~/repo/azul1/terraform $ cd ..
(.venv) daniel@Crispin ~/repo/azul1 $ CI_COMMIT_REF_NAME=anvilprod make deploy
/Library/Developer/CommandLineTools/usr/bin/make -C lambdas

…

Terraform will perform the following actions:

  # aws_cloudwatch_dashboard.dashboard will be updated in-place
…
  # aws_cloudwatch_log_group.error_log will be destroyed
  # (because aws_cloudwatch_log_group.error_log is not in configuration)
…
  # aws_cloudwatch_log_group.index_log will be destroyed
  # (because aws_cloudwatch_log_group.index_log is not in configuration)
…
  # aws_cloudwatch_log_group.search_log will be destroyed
  # (because aws_cloudwatch_log_group.search_log is not in configuration)
…
  # aws_cloudwatch_log_resource_policy.index will be destroyed
  # (because aws_cloudwatch_log_resource_policy.index is not in configuration)
…
  # aws_elasticsearch_domain.index will be destroyed
  # (because aws_elasticsearch_domain.index is not in configuration)
…
  # aws_iam_role_policy.indexer will be updated in-place
…
  # aws_iam_role_policy.service will be updated in-place
…
  # aws_lambda_function.indexer will be updated in-place
…
  # aws_lambda_function.indexer_aggregate will be updated in-place
…
  # aws_lambda_function.indexer_aggregate_retry will be updated in-place
…
  # aws_lambda_function.indexer_contribute will be updated in-place
…
  # aws_lambda_function.indexer_contribute_retry will be updated in-place
…
  # aws_lambda_function.indexer_indexercachehealth will be updated in-place
…
  # aws_lambda_function.service will be updated in-place
…
  # aws_lambda_function.service_manifest will be updated in-place
…
  # aws_lambda_function.service_servicecachehealth will be updated in-place
…
  # aws_lambda_layer_version.dependencies must be replaced
…
  # aws_security_group.elasticsearch will be destroyed
  # (because aws_security_group.elasticsearch is not in configuration)
…

Plan: 1 to add, 12 to change, 7 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: no

Apply cancelled.
```

--- a/UPGRADING.rst
+++ b/UPGRADING.rst
@@ -20,6 +20,25 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#6218 Delete hammerbox ES domain
+================================
+
+Operator
+~~~~~~~~
+
+Due to an open issue with the `Terraform provider opensearch`_, the ``deploy``
+job will fail on ``hammerbox`` when building the feature branch. After this
+occurs, run the following commands::
+
+    _select hammerbox
+    cd $project_root/terraform
+    terraform state rm opensearch_cluster_settings.index
+
+Then, retry the ``deploy`` job on GitLab. It should now succeed.
+
+.. _Terraform provider opensearch: https://github.com/opensearch-project/terraform-provider-opensearch/issues/60
+
+
 DataBiosphere/azul-private#6 data-browser: Content Security Policy (CSP) Not Implemented
 ========================================================================================
 

--- a/deployments/anvilprod/environment.py
+++ b/deployments/anvilprod/environment.py
@@ -761,9 +761,9 @@ def env() -> Mapping[str, Optional[str]]:
 
         'AZUL_ENABLE_MONITORING': '1',
 
-        # $0.382/h × 4 × 24h/d × 30d/mo = $1100.16/mo
+        # $0.382/h × 6 × 24h/d × 30d/mo = 1,650.24/mo
         'AZUL_ES_INSTANCE_TYPE': 'r6gd.xlarge.elasticsearch',
-        'AZUL_ES_INSTANCE_COUNT': '4',
+        'AZUL_ES_INSTANCE_COUNT': '6',
 
         'AZUL_CONTRIBUTION_CONCURRENCY': '300/64',
 

--- a/deployments/hammerbox/environment.py
+++ b/deployments/hammerbox/environment.py
@@ -786,15 +786,13 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DUOS_SERVICE_URL': 'https://consent.dsde-prod.broadinstitute.org',
         'AZUL_TERRA_SERVICE_URL': 'https://firecloud-orchestration.dsde-prod.broadinstitute.org',
 
+        # Personal deployments & `hammerbox` share an ES domain with `anvilprod`
+        'AZUL_SHARE_ES_DOMAIN': '1',
+        'AZUL_ES_DOMAIN': 'azul-index-anvilprod',
+
         **(
             {
-                # $0.382/h × 2 × 24h/d × 30d/mo = $550.08/mo
-                'AZUL_ES_INSTANCE_TYPE': 'r6gd.xlarge.elasticsearch',
-                'AZUL_ES_INSTANCE_COUNT': '4',
             } if is_sandbox else {
-                # Personal deployments share an ES domain with `hammerbox`
-                'AZUL_SHARE_ES_DOMAIN': '1',
-                'AZUL_ES_DOMAIN': 'azul-index-hammerbox',
                 # Personal deployments use fewer Lambda invocations in parallel.
                 'AZUL_CONTRIBUTION_CONCURRENCY': '8',
                 'AZUL_AGGREGATION_CONCURRENCY': '8',

